announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements.vm

```diff
@@ -251,6 +251,7 @@
 						#foreach ($ann_item in $showMessagesList)
 							#set ($rowCount =$rowCount + 1)
 							#set($ann_item_props=$ann_item.getProperties())
+							#set($isHightlighted = false)
```

Comment:
I feel we are defining the variable twice, could be maybe make a comparison?

Maybe something like this? 

#set($isHightlighted = (true == $ann_item_props.getProperty("highlight")) )

or maybe as string?

#set($isHightlighted = ("true".equals($ann_item_props.getProperty("highlight")) ) )

I mean, I don't have anything against this change, but I feel we're defining the variable two times.

Reply:
Ok. I will try